ws-security-dom/src/main/java/org/apache/wss4j/dom/message/WSSecEncryptedKey.java

```diff
@@ -389,7 +398,15 @@ protected void createEncryptedKeyElement(X509Certificate remoteCert, Crypto cryp
             }
             encryptedKeyElement.appendChild(keyInfoElement);
         }
+    }
 
+    /**
+     * Method verifies is the key agreement method is not empty
+     * @param keyAgreementMethod the key agreement method
+     * @return true if the key agreement method is not empty else false
+     */
+    private boolean isKeyAgreementMethodNotEmpty(String keyAgreementMethod) {
```

Comment:
Change it to isKeyAgreementConfigured ?

Reply:
Renamed function to isKeyAgreementConfigured